receiver/vmmetricsreceiver/vm_metrics_constants.go

```diff
@@ -15,131 +15,88 @@
+// VM and process metric constants.
 
-var mRuntimeAllocMem = stats.Int64("process/memory_alloc", "Number of bytes currently allocated in use", "By")
-var viewAllocMem = &view.View{
-	Name:        mRuntimeAllocMem.Name(),
-	Description: mRuntimeAllocMem.Description(),
-	Measure:     mRuntimeAllocMem,
-	Aggregation: view.LastValue(),
-	TagKeys:     nil,
+var metricAllocMem = &metricspb.MetricDescriptor{
+	Name:        "process/memory_alloc",
+	Description: "Number of bytes currently allocated in use",
+	Unit:        "By",
+	Type:        metricspb.MetricDescriptor_GAUGE_INT64,
+	LabelKeys:   nil,
 }
 
-var mRuntimeTotalAllocMem = stats.Int64("process/total_memory_alloc", "Number of allocations in total", "By")
-var viewTotalAllocMem = &view.View{
-	Name:        mRuntimeTotalAllocMem.Name(),
-	Description: mRuntimeTotalAllocMem.Description(),
-	Measure:     mRuntimeTotalAllocMem,
-	Aggregation: view.LastValue(),
-	TagKeys:     nil,
+var metricTotalAllocMem = &metricspb.MetricDescriptor{
+	Name:        "process/total_memory_alloc",
+	Description: "Number of allocations in total",
+	Unit:        "By",
+	Type:        metricspb.MetricDescriptor_GAUGE_INT64,
+	LabelKeys:   nil,
 }
 
-var mRuntimeSysMem = stats.Int64("process/sys_memory_alloc", "Number of bytes given to the process to use in total", "By")
-var viewSysMem = &view.View{
-	Name:        mRuntimeSysMem.Name(),
-	Description: mRuntimeSysMem.Description(),
-	Measure:     mRuntimeSysMem,
-	Aggregation: view.LastValue(),
-	TagKeys:     nil,
+var metricSysMem = &metricspb.MetricDescriptor{
+	Name:        "process/sys_memory_alloc",
+	Description: "Number of bytes given to the process to use in total",
+	Unit:        "By",
+	Type:        metricspb.MetricDescriptor_GAUGE_INT64,
+	LabelKeys:   nil,
 }
 
-var mCPUSeconds = stats.Float64("process/cpu_seconds", "CPU seconds for this process", "s")
-var viewCPUSeconds = &view.View{
-	Name:        mCPUSeconds.Name(),
-	Description: mCPUSeconds.Description(),
-	Measure:     mCPUSeconds,
-	Aggregation: view.Sum(),
-	TagKeys:     nil,
+var metricProcessCPUSeconds = &metricspb.MetricDescriptor{
+	Name:        "process/cpu_seconds",
+	Description: "CPU seconds for this process",
+	Unit:        "s",
+	Type:        metricspb.MetricDescriptor_CUMULATIVE_DOUBLE,
+	LabelKeys:   nil,
 }
 
-var mUserCPUSeconds = stats.Float64("system/cpu_seconds/user", "Total kernel/system user CPU seconds", "s")
-var viewUserCPUSeconds = &view.View{
-	Name:        mUserCPUSeconds.Name(),
-	Description: mUserCPUSeconds.Description(),
-	Measure:     mUserCPUSeconds,
-	Aggregation: view.Sum(),
-	TagKeys:     nil,
+var metricCPUSeconds = &metricspb.MetricDescriptor{
+	Name:        "system/cpu_seconds",
+	Description: "Total kernel/system CPU seconds broken down by different stages",
+	Unit:        "s",
+	Type:        metricspb.MetricDescriptor_CUMULATIVE_DOUBLE,
+	LabelKeys:   []*metricspb.LabelKey{{Key: "stage", Description: "Stage of CPU time, e.g user/system/idle"}},
```

Comment:
nit: would state be better than stage?
idle state
iowait state
running state (kernel vs user)

Reply:
SG, updated.